CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to WordPress-iOS
+
+Hi! Thank you for your interest in contributing to the app, we really appreciate it.
+
+There are many ways to contribute – reporting bugs, feature suggestions, fixing bugs, submitting pull requests for enhancements.
+
+## Reporting Bugs, Asking Questions, Sending Suggestions
+
+Just [file a GitHub issue](https://github.com/wordpress-mobile/WordPress-iOS/issues), that’s all. If you want to prefix the title with a “Question:”, “Bug:”, or the general area of the application, that would be helpful, but by no means mandatory. If you have write access, add the appropriate labels.
+
+If you’re filing a bug, specific steps to reproduce are helpful. Please include the part of the app that has the bug, along with what you expected to see and what happened instead.  A screenshot is always helpful as well.
+
+## Submitting code changes
+
+We welcome Pull Requests for any new feature or bug fix that you'd like to contribute!  More details on our process can be found in the [Make WordPress Mobile handbook](https://make.wordpress.org/mobile/handbook/pathways/ios/), specifically [here](https://make.wordpress.org/mobile/handbook/pathways/ios/how-to-contribute/).
```

Comment:
Yeah I would have to agree with this. Can we put something about if it's an untracked issue, they should first open an issue with some mention of how they would like to approach it?

Related:
- What about the .org slack? Are people in there? Is that somewhere we can be more active?
- Should we be more open about our roadmap? What would that look like?